raspi/controlserver/controlserver.py

```diff
@@ -20,8 +21,20 @@
 mountPath = '/media'
 
 wifi_search_folder = os.path.join(dir_path, '../access_point')
-susiconfig = '/home/pi/SUSI.AI/bin/susi-config'
 
+sys.path.insert(0, dir_path + "/../../pythonmods")
```

Comment:
This should not be necessary, since we start the control server from the systemd unit file with 
```
[Service]
Type=simple
WorkingDirectory=/home/pi/SUSI.AI
Environment=PYTHONPATH=/home/pi/SUSI.AI/susi_installer/pythonmods
ExecStart=/usr/bin/python3 /home/pi/SUSI.AI/susi_installer/raspi/controlserver/controlserver.py
```
so the PYTHONPATH is already set properly. If you need to test, just locally link the susi_config directory to the control server directory like we do with the others

Reply:
Thank you, I have updated it.